Source/Fuse.Controls.Native/iOS/View.uno

```diff
@@ -11,11 +12,66 @@ namespace Fuse.Controls.Native.iOS
 
 		readonly ObjC.Object _handle;
 
-		protected View(ObjC.Object handle) : this(handle, false) {}
+		protected View(ObjC.Object handle, ViewHandle.InputMode inputmode = ViewHandle.InputMode.Automatic) : this(handle, false, inputmode) {}
 
-		protected View(ObjC.Object handle, bool isLeafView) : base(handle, isLeafView)
+		protected View(ObjC.Object handle, bool isLeafView, ViewHandle.InputMode inputmode = ViewHandle.InputMode.Automatic) : base(handle, isLeafView, inputmode)
 		{
 			_handle = handle;
 		}
+
+		protected void DispatchTouchesBegan(Visual origin, ObjC.Object touches, [CallerFilePath] string filePath = "", [CallerLineNumber] int lineNumber = 0)
+		{
+			if (ErrorCheck(touches, filePath, lineNumber))
+				InputDispatch.OnTouchesBegan(origin, touches);
+		}
+
+		protected void DispatchTouchesMoved(Visual origin, ObjC.Object touches, [CallerFilePath] string filePath = "", [CallerLineNumber] int lineNumber = 0)
+		{
+			if (ErrorCheck(touches, filePath, lineNumber))
+				InputDispatch.OnTouchesMoved(origin, touches);
+		}
+
+		protected void DispatchTouchesEnded(Visual origin, ObjC.Object touches, [CallerFilePath] string filePath = "", [CallerLineNumber] int lineNumber = 0)
+		{
+			if (ErrorCheck(touches, filePath, lineNumber))
+				InputDispatch.OnTouchesEnded(origin, touches);
+		}
+
+		protected void DispatchTouchesCancelled(Visual origin, ObjC.Object touches, [CallerFilePath] string filePath = "", [CallerLineNumber] int lineNumber = 0)
+		{
+			if (ErrorCheck(touches, filePath, lineNumber))
+				InputDispatch.OnTouchesCancelled(origin, touches);
+		}
```

Comment:
I can't find any call-sites for any of these `DispatchTouches*`-methods... What's up, how can any of this stuff even work?